sumpy/kernel.py

```diff
@@ -662,44 +664,62 @@ def get_pde_as_diff_op(self):
         return (laplacian(w) - lam**2 * w)
 
 
-class StokesletKernel(ExpressionKernel):
-    init_arg_names = ("dim", "icomp", "jcomp", "viscosity_mu_name")
+class ElasticityKernel(ExpressionKernel):
+    init_arg_names = ("dim", "icomp", "jcomp", "viscosity_mu_name",
+            "poisson_ratio")
 
-    def __init__(self, dim, icomp, jcomp, viscosity_mu_name="mu"):
+    def __new__(cls, dim, icomp, jcomp, viscosity_mu="mu", poisson_ratio="nu"):
+        if poisson_ratio == "0.5":
```

Comment:
Is a string really the best solution here? Could you explain?

Reply:
No, it isn't. I tried pymbolic expressions, but it fails with,
```
TypeError: unsupported type for persistent hash keying: <class 'pymbolic.primitives.Variable'>
```
for `mu`.

Reply:
Thanks for making this happen.